### ISSUE TITLE
Shall we migrate recipes to conda-forge?

### ISSUE THREAD

**Issue by rmax** [scrapinghub/scrapinghub-conda-recipes#37]
Conda-forge channel is a community-supported channel with a huge list of packages. We could leverage it's popularity and have scrapy there.

**Comment by rmax** [comment]
CC @redapple

**Comment by redapple** [comment]
Looks good @rolando .
Can you take care of this?

**Comment by rmax** [comment]
@redapple yes. The only details I see are:
- The install command will change to `conda install -c conda-forge scrapy`
- I can take a few days or weeks to get our recipe added: https://github.com/conda-forge/staged-recipes/pulls (but afterwards it gets its own repository)

So, I think we can make the PRs to conda-forge, and meanwhile update our package for scrapy 1.1.1.

**Comment by redapple** [comment]
Sounds like a plan! Thanks @rolando

**Comment by redapple** [comment]
@rolando , has this task been started, to your knowledge?

**Comment by rmax** [comment]
@redapple no, if there is somebody else who could start this right away that'd be great. Otherwise I can take it over the weekend.

**Comment by rmax** [comment]
@redapple  I just went ahead and added the PR: https://github.com/conda-forge/staged-recipes/pull/1646

I added @redapple and @kmike as maintainers, besides myself (the ones who will get notifications, PRs, etc). I hope that is OK to you :)

Let me know if we should add anyone else to the maintainers list.
